integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
+      const tmpFile = tmp.fileSync()
+      const expectedError = 'Row length does not match headers'
+      const csvFilePath = path.join(samplesFolder, 'faulty-sample.csv')
+
+      // eslint-disable-next-line max-len
+      exec(`${binPath} -p ${projectKey}  -i ${csvFilePath} --logFile ${tmpFile.name}`,
+        (error, stdout, stderr) => {
+          expect(error).toBeTruthy()
+          expect(stderr).toMatch(expectedError)
+
+          fs.readFile(tmpFile.name, { encoding: 'utf8' }, (err, data) => {
+            expect(data).toContain(expectedError)
+            tmpFile.removeCallback()
```

Comment:
This is not necessary as you did not pass the keep option on line `121`
See [here](https://www.npmjs.com/package/tmp#synchronous-file-creation)